### DISCUSSION TITLE
[错误报告]: 刷新推荐页面后，cpu占用过高，会导致程序假死一段时间

### DISCUSSION THREAD

**Discussion by wintekk** [jxxghp/MoviePilot#3344] [讨论]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

1.9.16

### 运行环境

Docker

### 问题类型

主程序运行问题

### 问题描述

刷新推荐页面后，cpu占用过高，会飙升到120%-150%，查看日志后发现，刷新推荐页面后，后台一直在请求豆瓣api，会导致程序假死一段时间，点击其他页面无反应，等待豆瓣api请求完成后，后台cpu占用恢复正常。网络环境是正常的，豆瓣能正常链接
<img width="685" alt="屏幕截图 2024-09-14 001213" src="https://github.com/user-attachments/assets/d6302cee-2441-41cd-b7a7-08be4b23b66f">
<img width="837" alt="屏幕截图 2024-09-14 001316" src="https://github.com/user-attachments/assets/1c869397-b3a0-43e1-addf-63ac4f20443f">
<img width="278" alt="屏幕截图 2024-09-14 001551" src="https://github.com/user-attachments/assets/2a135bb1-2f72-46fb-b6e8-fcf3ec09f414">

**Comment by wintekk** [comment]
出问题时的后台日志
[moviepilot.csv](https://github.com/user-attachments/files/16996836/moviepilot.csv)

**Comment by LONGHEIHEI** [comment]
v2 最新版也有这个问题

**Comment by linkq** [comment]
同样问题，最新版本2.08
